utils/cache/cache.h

```diff
@@ -176,17 +176,38 @@ status_t cache_list_size(const char* const key, int* len);
 status_t cache_list_exist(const char* const key, const char* const value, const int value_len, bool* exist);
 
 /**
- * Pop an element from the list in in-memory cache
+ * @brief Pop an element from the list in in-memory cache
  *
- * @param[in] key Key for key-value starage
- * @param[out] res The element with assigning index
+ * @param key[in] Key for key-value starage
+ * @param res[out] The element with assigned index
  *
  * @return
  * - SC_OK on success
  * - non-zero on error
  */
 status_t cache_list_pop(const char* const key, char* res);
 
+/**
+ * @brief Get how many bytes redis has occupied.
+ *
+ * @return
+ * - Used size of the cache database on success
+ * - `-1` on error
+ */
+long int cache_occupied_space();
+
+/**
+ * @brief Turn off the Caching service.
+ *
+ */
+void cache_server_lock();
```

Comment:
Why would you use `lock` and `unlock` for switching caching service?

Reply:
There are three kinds of redis lists are used in buffering transactions. Cross lists operation may cause data racing, that is why we need an operation similar to (un)lock here.

Reply:
Can we simply use enable/disable for naming?

Reply:
sure